src/v2/guide/single-file-components.md

```diff
@@ -52,20 +52,22 @@ Incluso si no le gusta la idea de utilizar Single-File Components, usted puede a
 
 ### Ejemplo con Sandbox
 
-Si quieres profundizar y comenzar a jugar con single-file components, visita [esta aplicación to-do de ejemplo ](https://codesandbox.io/s/o29j95wx9) en CodeSandbox.
+Si quieres profundizar y comenzar a jugar con Componentes de un Solo Archivo, visita [esta aplicación to-do de ejemplo](https://codesandbox.io/s/o29j95wx9) en CodeSandbox.
 
-### Para usuarios nuevos en Sistemas de Construcción de Módulos en JavaScript
+### Para usuarios nuevos en Sistemas de Compilación de Módulos en JavaScript
 
-Con componentes `.vue`, estamos entrando en el reino de aplicaciones JavaScript avanzadas. Eso significa aprender a usar algunas herramientas adicionales si aún no lo ha hecho:
+Con los componentes `.vue` entramos en el reino de las aplicaciones JavaScript avanzadas. Esto significa aprender a usar algunas herramientas adicionales, si es que aún no lo has hecho:
 
-- **Node Package Manager (NPM)**: Lea la [Guía para comenzar](https://docs.npmjs.com/getting-started/what-is-npm) a través de a sección _10: Uninstalling global packages_.
+- **Manejador de Paquetes de Node** (Node Package Manager, NPM): Lee la Sección 10 de la [Guía para comenzar](https://docs.npmjs.com/getting-started/what-is-npm): _Desinstalando paquetes globales_ (_Uninstalling global packages_).
 
-- **Modern JavaScript with ES2015/16**: Lea la guía de Babel [Learn ES2015](https://babeljs.io/docs/learn-es2015/). Usted no tiene que memorizar cada característica ahora mismo, pero mantenga esta pagina como una referencia a la cual pueda regresar.
+- **JavaScript moderno con ES2015/16** (Modern JavaScript with ES2015/16): Lee la guía de Babel [Aprende ES2015](https://babeljs.io/docs/learn-es2015/). No es necesario memorizar cada característica, pero mantén esta página como una referencia a la cual regresar.
 
-Despues de que usted haya tomado un día para profundizar en estos recursos, le recomendamos revisar la plantilla [webpack](https://vuejs-templates.github.io/webpack). Sigue las instrucciones y debería tener un proyecto Vue con componentes `.vue`, ES2015, y hot-reloading en un instante.
-
-Para aprender más sobre Webpack, visite [su documentación oficial](https://webpack.js.org/configuration/) y [Webpack Academy](https://webpack.academy/p/the-core-concepts). En Webpack, cada archivo puede ser transformado por un "loader" antes de ser incluido en el the bundle, y Vue ofrece el plugin [vue-loader](https://vue-loader.vuejs.org) para traducir single file componentes (`.vue`).
+Luego de tomarte un día para profundizar en estos recursos, te recomendamos consultar [Vue CLI 3](https://cli.vuejs.org/). Sigue las instrucciones y deberías tener un proyecto Vue con componentes `.vue`, ES2015, y recarga en caliente en unos pocos minutos!
```

Comment:
Same here (about translating `hot-reload`)

Reply:
hot-reload it is